frontend/src/components/NavBarDropdown.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+import React, { useState } from "react";
+import { signOut } from "next-auth/react";
+import { LogOut, User } from "react-feather";
+
+export default function Dropdown(props: { initials: string }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  function toggle() {
+    setIsOpen((old) => !old);
+  }
+
+  const transClass = isOpen ? "flex" : "hidden";
+
+  return (
+    <>
+      <div className="relative">
+        <button
+          className="flex rounded-full border border-white h-9 w-9 overflow-hidden justify-center items-center"
```

Comment:
Er overflow-hidden nødvendig her?

Reply:
Var i det tilfellet at hvis man har mange navn at slutten forsvant i stedet for å gå ut av sirkelen. Jeg er åpen for å fjerne den, men siden jeg ikke fikk skriftstørrelse til å være dependent på lengden av navn ble dette løsningen min 😅

Reply:
Kanskje prøv noe lignende : ${props.initials.length > 3 && "text-sm"}

Reply:
Eller bytt til padding og justify-center + items-center

Reply:
Fikset i parprogrammering